Assets/Scripts/Defence/TowerController.cs

```diff
@@ -92,10 +90,6 @@ void DealDamageToEnemiesInRange()
                     enemyMovement.TakeDamage(damage);
                 }
             }
-            else
```

Comment:
Why is this gone?

Reply:
No idea, think it was being funky at the start, will put it back

Reply:
Done